src/components/DailyStats/DailyStats.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Card from "./Card/Card";
+import DataTable from './DataTable/DataTable';
+
+
+const DailyStats = ({data, type, yAccessor}) => {
+  return (
+    <div>
+      <Card data={data} type={type} yAccessor={yAccessor}/>
+      <DataTable data={data[0]} type={type}/> 
```

Comment:
Move Data table out of the Dailystats component and just add it to the bottom of the app.js page